src/raw.rs

```diff
@@ -160,6 +161,14 @@ impl Display for RawValue {
     }
 }
 
+impl PartialEq<RawValue> for RawValue {
+    fn eq(&self, other: &RawValue) -> bool {
+        self.json.eq(&other.json)
+    }
+}
+
+impl Eq for RawValue {}
```

Comment:
I don't think we should add these impls. What is your use case for them? It is potentially surprising that the same JSON object with only whitespace differences would compare unequal. Users who want this behavior should use `a.get() == b.get()` to compare the raw values as &amp;str.